ios/FluentUI/DrawerVnext/Drawer+Drag.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+
+internal extension Drawer {
```

Comment:
This will add a significant code when while addressing #356 . SwiftUI encourage modular structure as opposed monolithic views.

Reply:
You're not making it modular if your extension is specific to just the Drawer, are you?

Like I said before: if this will add significant amounts of code when you implement the gesture presentation, I can see the value for it.

To me this seems not needed for now.

Reply:
I have converted this to a private func, I will defer the need for gesture extension and add when required.